Makefile

```diff
@@ -155,18 +155,18 @@ define CROSSBUILD_HELP_INFO
 # cross build components.
 #
 # Args:
-#   WHAT: Component names to be lint check. support: $(CROSSBUILD_COMPONENTS)
+#   WHAT: Component names to be lint check. support: $(BINARIES)
```

Comment:
Use ${BINARIES} here, CROSSBUILD_COMPONENTS seems can be removed? Line L150

Reply:
Done. Have removed :)